web/client/components/map/leaflet/plugins/WMTSLayer.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright 2015, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const Layers = require('../../../../utils/leaflet/Layers');
+const CoordinatesUtils = require('../../../../utils/CoordinatesUtils');
+const WMSUtils = require('../../../../utils/leaflet/WMSUtils');
+const L = require('leaflet');
+const objectAssign = require('object-assign');
+const {isArray, isEqual} = require('lodash');
+const SecurityUtils = require('../../../../utils/SecurityUtils');
+
+
+L.TileLayer.WMTS = L.TileLayer.extend({

```

Comment:
I dont really understand what needs to be done in this part.
Do you mean to load all the const from an external file? Like import them?